japan/ood-2022/README.md

```diff
@@ -0,0 +1,34 @@
+[![Okinawa Open Days 2022 logo](images/logo-ood-2022.webp)](https://www.okinawaopendays.com/)<br>
+[![Okinawa Open Days 2022 title](images/title-ood-2022.webp)](https://www.okinawaopendays.com/)
+
+---
+
+# Okinawa Open Days 2022: Kubernetes Upstream Training
+
+Welcome to Kubernetes Upstream Training!!
+
+This is the location of [Kubernetes Upstream Training](https://www.okinawaopendays.com/session-dec14-training)(Dec 14th 2022) activities.
+
+## 資料 (Documents)
+
+* [参加者の皆様への宿題 Attendee prerequisites (in preparation)](../assets/attendee-prerequisites.md)
+* [スライド Slides](../assets/slide.pdf)
+* [Kubernetes contributors guide](https://github.com/kubernetes/community/tree/master/contributors/guide)
+
+## 講師（Facilitators）
+
+| Icon | Name | Slack | Affiliation | Focus on, Role.. |
+| ------------- | ------------- | ------------- | ------------- | ------------- |
+|<a href="https://github.com/oomichi"><img src="https://avatars.githubusercontent.com/u/1287154?s=50"></a>| <a href="https://github.com/oomichi">Kenichi Omichi</a> | @oomichi | NEC | - sig-testing<br> - Approver of k8s.io/kubernetes/test<br> - Owner of kubernetes-sigs/kubespray |
+|<a href="https://github.com/atoato88"><img src="https://avatars.githubusercontent.com/u/748740?s=50"></a>| <a href="https://github.com/atoato88">Akihito Inoh</a> | @akihito-inou(atoato88) | NEC Solution Innovators | - sig-cluster-lifecycle<br> - Approver of kubernetes-sigs/kubebuilder-declarative-pattern <br> - k/dashboard i18n ja |
+|<a href="https://github.com/shu-mutou"><img src="https://avatars.githubusercontent.com/u/12838129?s=50"></a>| <a href="https://github.com/shu-mutou">Shu Muto</a> | @Shu Muto | NEC Solution Innovators | - Chair of sig-ui<br> - Approver of k/dashboard |
+|<a href="https://github.com/YuikoTakada"><img src="https://avatars.githubusercontent.com/u/6335296?s=50"></a>| <a href="https://github.com/YuikoTakada">Yuiko Mouri</a> | @YuikoTakada | NEC Solution Innovators | - sig-storage<br> - sig-node |
+|<a href="https://github.com/ziyi-xie"><img src="https://avatars.githubusercontent.com/u/92832323?s=50"></a>| <a href="https://github.com/ziyi-xie">Xie Ziyi</a> | @XIE ZIYI | NEC Solution Innovators | - sig-docs<br> - k/dashboard i18n ja  |
+|<a href="https://github.com/s-kawamura-w664"><img src="https://avatars.githubusercontent.com/u/76156228?s=50"></a>| <a href="https://github.com/s-kawamura-w664">Shinji Kawamura</a> | @Shinji Kawamura (s-kawamura-w664)  | NEC Solution Innovators | - sig-node |
```

Comment:
```suggestion
|<a href="https://github.com/s-kawamura-w664"><img src="https://avatars.githubusercontent.com/u/76156228?s=50"></a>| <a href="https://github.com/s-kawamura-w664">Shinji Kawamura</a> | @Shinji Kawamura (s-kawamura-w664)  | NEC Solution Innovators | - sig-docs<br> - sig-node |
```
Could you please add sig-docs in my 'Focus on'.

Reply:
I've merged your suggestion, thank you.